Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -65,7 +65,7 @@ def self.audit_args
              description: "Prefix every line of output with the file or formula name being audited, to " \
                           "make output easy to grep."
       switch "--display-failures-only",
-             description: "Only display casks that fail the audit. This is the default for formulae."
+             description: "Only display casks that fail the audit. This is the default for formulae and casks."
```

Comment:
```suggestion
             description: "Only display casks that fail the audit. This is the default for formulae and casks.",
             hidden: true
```

IMO if we're deprecating it